app/controllers/volunteer_ops_controller.rb

```diff
@@ -75,6 +92,27 @@ def build_map_markers(organisations)
     end
   end
 
+  # This needs to be a helper method for the do-it API
+  def collect_all_items(host, href, orgs)
+    n = 1
+    while href
+      url = host + href
+      response = HTTParty.get(url)
+      if response.body && response.body != '[]'
+        resp_items = JSON.parse(response.body)['data']['items']
+        resp_items.each do |item|
+          org = OpenStruct.new(latitude: item['lat'], longitude: item['lng'], name: item['title'], id: n,
```

Comment:
Metrics/LineLength: Line is too long. [105/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)